test/remote/gateways/remote_braintree_blue_test.rb

```diff
@@ -1300,6 +1300,10 @@ def test_unsuccessful_credit_card_purchase_and_return_payment_details
     assert_equal 'credit_card', response.params['braintree_transaction']['payment_instrument_type']
     assert_equal 'Unknown', response.params['braintree_transaction']['credit_card_details']['prepaid']
     assert_equal 'Unknown', response.params['braintree_transaction']['credit_card_details']['debit']
+    assert_equal 'M', response.params.dig('braintree_transaction', 'cvv_response_code')
+    assert_equal 'I', response.params.dig('braintree_transaction', 'avs_response_code')
+    assert_equal 'Call Issuer. Pick Up Card.', response.params.dig('braintree_transaction', 'gateway_message')
+    assert_equal 'Unknown', response.params.dig('braintree_transaction', 'credit_card_details', 'country_of_issuance')
     assert_equal 'Unknown', response.params['braintree_transaction']['credit_card_details']['issuing_bank']
```

Comment:
Looks good, but just wondering why you added these assertions only to the unsuccessful purchase?

Reply:
Felt redundant to add them elsewhere 🤷🏽‍♂️

Reply:
@aenand I guess I just default to adding these kinds of things to a successful test because we are hoping there would be more successes than failures. 😄  It didn't seem like these would be specific to a failure scenario which is why I asked.